lesson-7/7lesson_dz.py

```diff
@@ -0,0 +1,26 @@
+phone_list = {
+    "tom": "+123456",
+    "bob": "+098765",
+    "alice": "+657483",
+    "dilan": "+990888",
+    "adam": "+344868",
+    "dora": "+776445",
+    "chuck": "+611234"
+}
+user_input = input('enter a command: ') # commands: stats, add, delete <name>, list, show <name>
+if user_input =='stats':
+    print(len(phone_list))
+elif user_input =='add':
+    name_phone_input = input('enter name and phone ')
+    split_input = name_phone_input.split()
+    name = split_input[0]
+    phone = split_input[1]
+    phone_list[name] = phone
```

Comment:
It is not allowed to override records. Please check if the key already exists and add a new record only if it doesn't.

Reply:
done